From a0a5af8e9d1468dda3005976bc5541d3a4c3fcb4 Mon Sep 17 00:00:00 2001
From: Yuan Si <do4suki@gmail.com>
Date: Fri, 29 Dec 2023 21:04:07 +0800
Subject: [PATCH] Revert "Adding hover state for brightness slider in
 quick settings"

This reverts commit 5bbf2af062ddd56934b81496fff746b41b2204df.

Change-Id: I3949cbda7b55129ceae933a3d02f84eb880df302
Signed-off-by: Yuan Si <do4suki@gmail.com>
---
 .../color/brightness_slider_overlay_color.xml | 22 -------------------
 .../brightness_progress_full_drawable.xml     |  7 ------
 .../quick_settings_brightness_dialog.xml      |  1 -
 3 files changed, 30 deletions(-)
 delete mode 100644 packages/SystemUI/res/color/brightness_slider_overlay_color.xml

diff --git a/packages/SystemUI/res/color/brightness_slider_overlay_color.xml b/packages/SystemUI/res/color/brightness_slider_overlay_color.xml
deleted file mode 100644
index a8abd793bd00..000000000000
--- a/packages/SystemUI/res/color/brightness_slider_overlay_color.xml
+++ /dev/null
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-  ~ Copyright (C) 2023 The Android Open Source Project
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~      http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:state_pressed="true" android:color="?attr/onShadeActive" android:alpha="0.12" />
-    <item android:state_hovered="true" android:color="?attr/onShadeActive" android:alpha="0.09" />
-    <item android:color="@color/transparent" />
-</selector>
\ No newline at end of file
diff --git a/packages/SystemUI/res/drawable/brightness_progress_full_drawable.xml b/packages/SystemUI/res/drawable/brightness_progress_full_drawable.xml
index c180fefabee4..4d9188c40822 100644
--- a/packages/SystemUI/res/drawable/brightness_progress_full_drawable.xml
+++ b/packages/SystemUI/res/drawable/brightness_progress_full_drawable.xml
@@ -26,13 +26,6 @@
             <corners android:radius="@dimen/rounded_slider_corner_radius"/>
         </shape>
     </item>
-    <item>
-        <shape>
-            <corners android:radius="@dimen/rounded_slider_corner_radius" />
-            <size android:height="@dimen/rounded_slider_height" />
-            <solid android:color="@color/brightness_slider_overlay_color" />
-        </shape>
-    </item>
     <item
         android:id="@+id/slider_icon"
         android:gravity="center_vertical|right"
diff --git a/packages/SystemUI/res/layout/quick_settings_brightness_dialog.xml b/packages/SystemUI/res/layout/quick_settings_brightness_dialog.xml
index 42ae4b7193b3..df8a56fa0845 100644
--- a/packages/SystemUI/res/layout/quick_settings_brightness_dialog.xml
+++ b/packages/SystemUI/res/layout/quick_settings_brightness_dialog.xml
@@ -36,6 +36,5 @@
             android:paddingEnd="0dp"
             android:progressDrawable="@drawable/brightness_progress_drawable"
             android:splitTrack="false"
-            android:clickable="true"
         />
     </com.android.systemui.settings.brightness.BrightnessSliderView>
-- 
2.43.0

